### ISSUE TITLE
Websocket proxy in port mappings

### ISSUE THREAD

**Issue by etursunbaev** [subutai-io/agent#917]
Need to add these nginx directives to templates
```    
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
```
to websocket proxying
https://www.nginx.com/blog/websocket-nginx/ additional info can be found here

**Comment by lbthomsen** [comment]
And how will that influence non-websocket connections?

**Comment by etursunbaev** [comment]
```
        proxy_set_header   Upgrade $http_upgrade;
        proxy_set_header   Connection $http_connection;
```
As discussed internally better add these lines to all templates and should not hurt anything

**Comment by marcoarthur** [comment]
Hello folks, I just hit this issue trying to understand why my deploy to subutai of Convos (http://convos.by/) that depends on WebSockets was failing to keep the connections up. But, these configurations didn't allow me to run it properly I had to add this line on the subutai-nginx configuration created by agent:
```
 proxy_http_version 1.1
```
After search for a while (https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_http_version).As I read the docs referred, it states when http proxying the proxy is by default set to http 1.0. And that was the problem I was getting with this application. Thks all. :)

**Comment by akarasulu** [comment]
Good catch @marcoarthur. @etursunbaev was right. Perhaps the fix never made it out the door into production.

**Comment by Dilshat** [comment]
@marcoarthur hi
I pushed your fix to DEV branch.
You can test with dev peer

**Comment by marcoarthur** [comment]
Thanks @Dilshat ! Will setup a dev peer and give you feedback ASAP

**Comment by marcoarthur** [comment]
@Dilshat sorry couldn't really test it yet, without a sandbox env by the moment.

**Comment by Dilshat** [comment]
@marcoarthur np , take your time !

**Comment by Dilshat** [comment]
@marcoarthur hey buddy
Did you have a chance to test the issue?
